### DISCUSSION TITLE
Colorize chart background based on thresholds

### DISCUSSION THREAD

**Discussion by mzakharo** [RomRider/apexcharts-card#500] [Q&A]
Hello,  

I have a value that I need to keep in range. Any way to colorize the background of the graph to have a 'green middle'?

Here is best I could do with Apexcharts vs mini-graph-card (which has issues with graph not being in sync with the value).
![image](https://user-images.githubusercontent.com/567867/218605362-e6447384-36c8-44bf-93f1-d71bb53f495b.png)

Here is what ideal graph would look like (grafana):
![image](https://user-images.githubusercontent.com/567867/218606143-e9fad143-7d7c-498c-ad99-55641c6b794a.png)

Any ideas on how to display/graph with values that need to be kept within a 'valid' range would also be helpful.

**Comment by Kertz1954** [comment]
Post your question again at this URL. Then I can give you an answer / formatted code. 
[https://community.home-assistant.io/t/apexcharts-card-a-highly-customizable-graph-card/272877/2652](url)

**Comment by adriel** [comment]
Did you ever find out how to do this?
